docs/deploy-with-docker.md

```diff
@@ -38,6 +38,10 @@ $ docker build -t apisix-dashboard:$tag .
 
 # For users in mainland China, the `ENABLE_PROXY` parameter can be provided to speed up module downloads.
 $ docker build -t apisix-dashboard:$tag . --build-arg ENABLE_PROXY=true
+
+# If you want to use the latest code to build, you can specify the `APISIX_DASHBOARD_VERSION` parameter to `master`. 
```

Comment:
codes

Reply:
@juzhiyuan do you mean to change `code` to `codes`?

Reply:
Yes, it's a grammar issue.

Reply:
fixed.